lib/metalsmith-unexpected-markdown.js

```diff
@@ -61,7 +53,9 @@ module.exports = function plugin(pluginOptions) {
                         async.waterfall([
                             function (cb) {
                                 debug('generating html for: %s', file);
-                                markdown.toHtml(options).then(function (html) {
+
+                                var content = data.contents.toString();
+                                new UnexpectedMarkdown(content).toHtml(options).then(function (html) {
```

Comment:
Re my comment (https://github.com/unexpectedjs/metalsmith-unexpected-markdown/pull/7#pullrequestreview-337890218), I thinl this is where we need to avoid calling `.toHtml()` if we've just updated examples.

Reply:
I'm sure you are right, but we need to fix the context isolation in unexpected-markdown.